spyder/utils/introspection/jedi_patch.py

```diff
@@ -22,10 +22,11 @@ def apply():
 
     See [1] and [2] module docstring."""
     from spyder.utils.programs import is_module_installed
-    if is_module_installed('jedi', '=0.9.0'):
+    if (is_module_installed('jedi', '=0.9.0') or
+            is_module_installed('jedi', '=0.10.0')):
```

Comment:
I remember that I wrote it that way so that when Jedi 0.11.0 comes out my patching won't thrash it(like it did when 0.10.0 came out).  I am also hoping not to have to patch 0.11.0  :-)

I can still make the change if you think that's best.